content/guides/migrating.md

```diff
@@ -266,13 +266,15 @@ It's no longer necessary to specify it in configuration.
 ```diff
 module: {
   rules: [
-    test: /.css$/,
--    loader: ExtractTextPlugin.extract("style-loader", "css-loader", { publicPath: "/dist" })
-+    loader: ExtractTextPlugin.extract({
-+      fallback: "style-loader",
-+      use: "css-loader",
-+      publicPath: "/dist"
-+    })
+    {
+      test: /.css$/,
+-      loader: ExtractTextPlugin.extract("style-loader", "css-loader", { publicPath: "/dist" })
++      loader: ExtractTextPlugin.extract({
```

Comment:
`use` is more powerful and it's the preferred option as we'll likely drop `loader` in a future release. That makes it a good recommendation.